DQMOffline/L1Trigger/src/L1TStage2CaloLayer2Offline.cc

```diff
@@ -774,5 +831,33 @@ bool L1TStage2CaloLayer2Offline::doesNotOverlapWithHLTObjects(const l1t::Jet & j
   return matchedObjects.empty();
 }
 
+void L1TStage2CaloLayer2Offline::endJob(){
+  normalise2DHistogramsToBinArea();
+}
+
+void L1TStage2CaloLayer2Offline::normalise2DHistogramsToBinArea() {
```

Comment:
In terms of changes, these are the only lines that introduce any change int the final histograms.
As @thomreis mentioned, they only refer to EG DQM and they've been tested locally.
However, the test-statistic is, of course, on the smaller side. Changes are verified on a quantitive scale but rather than statistically.